### PR TITLE
Path search: be more lenient and ignore extra whitespace around the `>` character

### PR DESCRIPTION
**Pull request by TheBestPessimist**

### CODE REVIEW

File: Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs [hunk 1]
```diff
@@ -196,31 +196,30 @@ private async Task<List<Result>> PathSearchAsync(Query query, CancellationToken
...
 
-            IAsyncEnumerable<SearchResult> directoryResult;
+            IAsyncEnumerable<SearchResult> userSelectionResult;
 
             var recursiveIndicatorIndex = query.Search.IndexOf('>');
 
             if (recursiveIndicatorIndex > 0 && Settings.PathEnumerationEngine != Settings.PathEnumerationEngineOption.DirectEnumeration)
             {
-                directoryResult =
+                userSelectionResult =
```
[the remark this answers is not in the document]
Comment (TheBestPessimist) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs:205]: Yes and no.

When you press <kbd>Tab</kbd> on a file, you get 2 entries: the normal `Use > to search this directory, enter to open containing directory` message and also the `file result`.
There is no actual folder where we are doing the search.

See here:
image


It seems like we are in a situation where your name makes sense to you, but not to me (maybe we need the opinion of a third person to settle this?).

When I was reading the code and I got to read the name `directoryResult`, I expected in there to be a list of directories, when in reality the name means "this holds the result of all the files and folders of a particular thing the user selected, the contents of 1. listing a file or 2. listing a folder".

I believe `userSelectionResult` is a name better explaining what we store in here: the resulting contents of a user selection. Hence the change.

[the remark this answers is not in the document]
Comment (TheBestPessimist) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs:205]: Lots of discussion here: https://discord.com/channels/727828229250875472/917500039931256933/1092434394880421958

**Review by taooceros — APPROVED** [review]

File: Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheBestPessimist) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs]: Resolution: revert the name change as it's a bikeshedding topic. 

I think my option is better, @taooceros thinks none of the options are better. In the end, it's simpler to revert and leave figuring out a new name for future us.

**Comment by TheBestPessimist** [comment]
@jjw24 this is not a bug, i'd say it's a feature. (you added label `bug`)

**Comment by jjw24** [comment]
Doesn't this trim call just remove the leading and trailing spaces of the query after `>` and those spaces shouldnt be considered during the search?

**Pull request merged**